src/util/io.c

```diff
@@ -147,8 +183,20 @@ socket_connect (
     struct addrinfo *p;
     char port_str[MAX_PORT_STR_LEN];
     int ret = 0;
-    char host_buff[_POSIX_HOST_NAME_MAX] __attribute__((unused));
+#ifdef _WIN32
+    char host_buff[_HOST_NAME_MAX];
+    const char *h = hostname;
+    WSADATA wsaData;
+    int iResult;
+    iResult = WSAStartup(MAKEWORD(2, 2), &wsaData);
+    if (iResult != 0) {
+        LOG_WARNING("WSAStartup failed: %d", iResult);
+        return TSS2_TCTI_RC_IO_ERROR;
+    }
+#else
+    char host_buff[HOST_NAME_MAX] __attribute__((unused));
```

Comment:
That should char host_buff[_HOST_NAME_MAX]

Reply:
Thanks, this is now fixed.